### DISCUSSION TITLE
Can I use beesworm graph for multi classification problem?

### DISCUSSION THREAD

**Discussion by saneehaAmir** [shap/shap#3120] [General]
Hello,

I am working on multi classification problem. I am dealing with 4 classes. I am able to plot summary bar graph showing the contribution of all four classes in prediction but I am unable to plot beesworm graph. I have tried but it raises the following error:

ValueError: The beeswarm plot does not support plotting explanations with instances that have more than one dimension!

Thanks for your cooperation.

**Comment by znacer** [comment]
Hi,
You can plot a beeswarm plot only for 1 class at a time.
You last line to produce beeswarm shall be something like : 
```py
class_to_show = 0
shap.plots.beeswarm(shap_values[:,:,class_to_show])
```
